README.md

```diff
@@ -70,12 +74,49 @@ From here click `Begin`, `Next` and `Authorise`.
 
 Currently you must SSH into the instance to apply the command but we will provide a remote exec in a future release.
 
+## Style Guidelines
+
+### Naming Convention
+
+Throughout the module, the naming convention used is as follows:
+- For Terraform resource names, the names consist of lowercase characters and _ to represent spaces in the name.
+- For AWS resource names, the names consist of lowercase characters and - to represent spaces in the name, with the <a name="input_prefix"></a> [prefix](#input\_prefix) variable attached to the beginning of the name and the <a name="input_environment"></a> [environment](#input\_environment).
+
+## Features
+
+### Best Practices
+
+#### Networking
+
+- This is a single layer architecture so only one availability zone and one subnet is created to host the virtual machine.
+- An internet gateway with the appropriate routes are created to enable that the virtual machine has full access to the internet.
+
+#### Security
+
+- Spot instance security group only allows [SSH](https://www.ucl.ac.uk/isd/what-ssh-and-how-do-i-use-it#:~:text=SSH%20or%20Secure%20Shell%20is,web%20pages)%20and%20share%20data.), [VNC](https://discover.realvnc.com/how-to-use-vnc-connect-remote-desktop-software#:~:text=VNC%20stands%20for%20Virtual%20Network,right%20in%20front%20of%20it.) and [RDP](https://www.techtarget.com/searchenterprisedesktop/definition/Remote-Desktop-Protocol-RDP#:~:text=What%20is%20remote%20desktop%20protocol,their%20physical%20work%20desktop%20computers.) inbound access from IP addresses specified by the user. This ensures that only specific IP addresses are allowed access. 
+- Spot instance security group allows full outbound access so that the instance has full access to the internet.
+- A key pair is generated to allow authentication when accessing the instance using port 22(SSH).
```

Comment:
it isnt generated. They generate the key pair. 

A user-provided SSH key is used to authenticate access to the instance through SSH.

Reply:
done